### PR TITLE
Fix pep8 and UT issues

### PR DESCRIPTION
**Pull request by nnDarshan**
tendrl-bug-id: Tendrl/ceph_integration#62
Signed-off-by: nnDarshan <darshan.n.2024@gmail.com>

### CODE REVIEW

**Comment by nnDarshan** [comment]
@r0h4n @nthomas-redhat @shtripat Please review

File: tendrl/ceph_integration/persistence/persister.py [hunk 1]
```diff
@@ -1,11 +1,22 @@
 from tendrl.ceph_integration.persistence.sync_objects import SyncObject
+from tendrl.commons.config import load_config
+from tendrl.commons.etcdobj.etcdobj import Server as etcd_server
 from tendrl.commons.persistence.etcd_persister import EtcdPersister
 
+config = load_config(
+    "gluster-integration",
+    "/etc/tendrl/tendrl.conf"
```
Comment (cybernth) [inline tendrl/ceph_integration/persistence/persister.py]: Might not work. tendrl.conf is gone and you will have module specific files. Also this function expected to load yaml files

**Review by cybernth — CHANGES_REQUESTED** [review]

File: tendrl/ceph_integration/persistence/persister.py (hunk 1 above)
Comment (r0h4n) [reply tendrl/ceph_integration/persistence/persister.py]: +1 @nnDarshan , this is a yaml loader, ensure you pass a yaml file path here

Comment (r0h4n) [reply tendrl/ceph_integration/persistence/persister.py]: And replace gluster with ceph

Comment (nnDarshan) [reply tendrl/ceph_integration/persistence/persister.py]: Done

File: tendrl/ceph_integration/manager/eventer.py [hunk 2]
```diff
@@ -23,9 +22,14 @@
 from tendrl.ceph_integration.types import OsdMap
 from tendrl.ceph_integration.types import ServiceId
 from tendrl.ceph_integration.util import now
+from tendrl.commons.config import load_config
 
 
-config = TendrlConfig()
+config = load_config(
+    "ceph-integration",
+    "/etc/tendrl/ceph-integration/ceph-integration.conf.yaml"
```
Comment (shtripat) [inline tendrl/ceph_integration/manager/eventer.py]: We are following the naming as `ceph-integration.yaml`

Comment (nnDarshan) [reply tendrl/ceph_integration/manager/eventer.py]: Done

**Review by shtripat — COMMENTED** [review]
LGTM

**Comment by r0h4n** [comment]
Please update this PR

**Pull request closed without merging**